### ISSUE TITLE
DeprecationWarning: Do not access Theme._vars, instead access the keys of Theme directly

### ISSUE THREAD

**Issue by ethrgeist** [timvink/mkdocs-git-revision-date-localized-plugin#145]
I'm using

- `mkdocs-git-revision-date-localized-plugin` version 1.2.8
- `mkdocs` version 1.6.1 
- `mkdocs-material`version 9.5.34

and i am getting the following deprecation warning:

```
INFO    -  DeprecationWarning: Do not access Theme._vars, instead access the keys of Theme directly.
             File "/some/random/pyenv/pypoetry/virtualenvs/sillyproject-py3.12/lib/python3.12/site-packages/mkdocs_git_revision_date_localized_plugin/plugin.py", line 81, in on_config
               theme_locale = custom_theme._vars.get("language")
             File "/some/random/pyenv/pypoetry/virtualenvs/sillyproject-py3.12/lib/python3.12/site-packages/mkdocs/theme.py", line 87, in _vars
               warnings.warn(
```

**Comment by timvink** [comment]
This should be fixed in #144 , making a new release now

**Comment by timvink** [comment]
Released: https://pypi.org/project/mkdocs-git-revision-date-localized-plugin/
